src/Orleans.Core/Timers/TimerManager.cs

```diff
@@ -0,0 +1,354 @@
+using System;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Orleans.Threading;
+
+namespace Orleans.Timers
+{
+    public static class TimerManager
```

Comment:
Shouldn't public surface be interface, so components using it can use mocks during testing?

Reply:
`Task.Delay` isn't an interface